site/docs/tutorial/android-app.md

```diff
@@ -349,43 +317,46 @@ symlinked from the following locations:
 * `$WORKSPACE/bazel-out`, which stores other types of build outputs
```

Comment:
`$WORKSPACE` does not make sense on Windows. I recommend some other notation, e.g. `<workspace>/bazel-out`.

Reply:
I removed the prefix, and added a note about where these directories can be found.